ui/src/app/subnets/views/VLANDetails/VLANSummary/VLANSummary.test.tsx

```diff
@@ -69,3 +89,20 @@ it("renders correct details", () => {
     generateLegacyURL(baseURLs.controller({ id: controller.system_id }))
   );
 });
+
+it("can display the edit form", () => {
+  const store = mockStore(state);
+  render(
+    <Provider store={store}>
+      <MemoryRouter>
+        <VLANSummary id={vlan.id} />
+      </MemoryRouter>
+    </Provider>
+  );
+  const button = screen.getByRole("button", { name: "Edit" });
+  expect(button).toBeInTheDocument();
+  expect(screen.queryAllByTestId("EditVLAN")).toHaveLength(0);
```

Comment:
```suggestion
  expect(screen.queryAllByRole("form", { name: "Edit VLAN" })).not.toBeInTheDocument()

Reply:
These still need to use `.toHaveLength(0);` because they're using `All` to find elements that don't exist (unless I'm missing something).

Reply:
Sorry, I meant `expect(screen.queryByRole("form", { name: "Edit VLAN" })).not.toBeInTheDocument()`.

Don't get me wrong, this is nitpicking. What you did is perfectly valid. `not.toBeInTheDocument()` is simply more meaningful and already used in other tests, would be good to be consistent.

Opened a PR with this change: https://github.com/canonical-web-and-design/maas-ui/pull/3621